src/inc/regdisp.h

```diff
@@ -369,11 +364,6 @@ inline void SyncRegDisplayToCurrentContext(REGDISPLAY* pRD)
 
 #if defined(_WIN64)
     pRD->SP         = (INT_PTR)GetSP(pRD->pCurrentContext);
-
-#ifdef DEBUG_REGDISPLAY
```

Comment:
@jkotas I wonder if there was some reason for using the CheckRegDisplaySP only for 64 bit processors.